uplink/piecestore/download.go

```diff
@@ -184,6 +190,11 @@ func (client *Download) Read(data []byte) (read int, err error) {
 			client.downloaded += int64(len(response.Chunk.Data))
 			client.unread.Fill(response.Chunk.Data)
 		}
+		// we got a piece hash and the original order limit (GET_REPAIR)
```

Comment:
I don't totally understand what this comment means

Reply:
If `response.Chunk` doesn't exist and `response.Hash` exists, it means the piecehash was sent over from `storagenode/piecestore/endpoint.go`. This only happens for a `GET_REPAIR`

Reply:
Do you think there is a better way to clarify this in the comment?